### PR TITLE
Implementation of Stack using queues in Python

### PR DESCRIPTION
**Pull request by DEEPALI-SUK**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Implementation of Stack using queues in Python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #2244

### CODE REVIEW

**Comment by DEEPALI-SUK** [comment]
Please guide me why these errors are coming, I changed only python files but then also it is showing errors in java.

File: Python/ds/Implementation_of_Stack_using_queues.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+class my_stack:
```
Comment (HarshCasper) [inline Python/ds/Implementation_of_Stack_using_queues.py:7]: Add a beginner-friendly pitch about the code

File: Python/ds/Implementation_of_Stack_using_queues.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    def __init__(self):
+        self.my_queue1 = my_queue()
+        self.my_queue2 = my_queue()
+    # push method takes an argument and enqueues it in my_queue1
+    # After enqueuing in my_queue1 every element of my_queue2 is
+    # dequeued and enqueued in my_queue1.
+    # The names of my_queue1 and my_queue2 are then swapped.
```
Comment (HarshCasper) [inline Python/ds/Implementation_of_Stack_using_queues.py]: Use multi-line string

File: Python/ds/Implementation_of_Stack_using_queues.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    # isempty method returns false if item_list is not empty else it returns true
+    def isempty(self):
+        return self.item_list == []
+
+ 
+if __name__ == '__main__':
+    object1 = my_stack()
+    print('Enter size')
+    n = int(input())
+    print('Enter n nubmers to be pushed into stack')
```
Comment (HarshCasper) [inline Python/ds/Implementation_of_Stack_using_queues.py]: Use a more descriptive console statement

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by DEEPALI-SUK** [comment]
@HarshCasper Kindly review it now, I have made the changes accordingly.

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @DEEPALI-SUK for code

**Comment by HarshCasper** [comment]
@all-contributors bot please add @DEEPALI-SUK for code

**Pull request merged**